app/src/main/java/com/rob729/quiethours/Activity/MainActivity.kt

```diff
@@ -29,7 +28,7 @@ class MainActivity : AppCompatActivity() {
         }
 
         val navController = myNavHostFragment.view?.let { Navigation.findNavController(it) }
-        NavigationUI.setupActionBarWithNavController(this, navController!!)
+//        NavigationUI.setupActionBarWithNavController(this, navController!!)
```

Comment:
Remove the line instead of commenting if it is not required anymore.

Reply:
Ok I will remove. It is not required since I am using custom toolbar and  app will crash if we use this condition